airflow/models/dag.py

```diff
@@ -2937,7 +2937,7 @@ def calculate_dagrun_date_fields(
 def dag(*dag_args, **dag_kwargs):
     """
     Python dag decorator. Wraps a function into an Airflow DAG.
-    Accepts kwargs for operator kwarg. Can be used to parametrize DAGs.
+    Accepts kwargs for operator kwarg. Can be used to parameterize DAGs.
```

Comment:
*Parametrize* seems more widespread? That’s the spelling Pytest uses as well. I don’t know.

Reply:
But not https://pypi.org/project/parameterized/ -- and this spelling was in the dictionary, the other wasn't.

(As a British English speaker param-et-er-ized is how I say it, so I was happy to go along with this.)